itkwidgets/viewer.py

```diff
@@ -323,6 +344,34 @@ async def get_current_scale(self):
         """
         return await self.viewer_rpc.itk_viewer.getLoadedScale()
 
+    @fetch_value
+    async def get_roi_image(self, scale: int = -1, name: str = 'Image') -> NgffImage:
+        """Get the image for the current ROI.
+
+        :param scale: scale of the primary image to get the slices for the
+        current roi. -1, the default, uses the current scale.
+        :type scale: int
+        :param name: Name of the loaded image data to use. 'Image', the
+        default, selects the first loaded image.
+        :type name:  str
+
+        :return: roi_image
+        :rtype:  NgffImage
+        """
+        roi_slices = await self.get_roi_slice(scale)
+        roi_region = await self.get_roi_region()
+        if store := self.stores.get(name):
+            multiscales = from_ngff_zarr(store)
+            loaded_image = multiscales.images[scale]
+            roi_data = loaded_image.data[roi_slices]
+            return to_ngff_image(
+                roi_data,
+                dims=loaded_image.dims,
+                scale=loaded_image.scale,
+                translation=roi_region[0]
```

Comment:
please also pass other kwargs, e.g. units

Reply:
Added in